examples/01_plotting/plot_3d_map_projection.py

```diff
@@ -0,0 +1,38 @@
+"""
+Making a surface plot of a 3D statistical map
+=============================================
+
+Download a statistical map from Neurovault and a cortical mesh (fsaverage),
```

Comment:
I would write "Download a 3D statistical map from". Because you said 3D data after.